src/VirtualClient/VirtualClient.Main/profiles/PERF-CPU-OPENSSL.json

```diff
@@ -1,117 +1,225 @@
+      }
+    },
+    {
+      "Type": "OpenSslExecutor",
+      "Parameters": {
+        "Scenario": "SHA256",
+        "CommandArguments": "speed -elapsed -seconds 100 sha256",
+        "PackageName": "openssl",
+        "Tags": "CPU,OpenSSL,Cryptography"
+      }
+    },
+    {
+      "Type": "OpenSslExecutor",
+      "Parameters": {
+        "Scenario": "SHA512",
+        "CommandArguments": "speed -elapsed -seconds 100 sha512",
+        "PackageName": "openssl",
+        "Tags": "CPU,OpenSSL,Cryptography"
+      }
+    },
+    {
+      "Type": "OpenSslExecutor",
+      "Parameters": {
+        "Scenario": "SHA3-256",
+        "CommandArguments": "speed -elapsed -seconds 100 sha3-256",
+        "PackageName": "openssl",
+        "Tags": "CPU,OpenSSL,Cryptography"
+      }
+    },
+    {
+      "Type": "OpenSslExecutor",
+      "Parameters": {
+        "Scenario": "SHA3-512",
+        "CommandArguments": "speed -elapsed -seconds 100 sha3-512",
+        "PackageName": "openssl",
+        "Tags": "CPU,OpenSSL,Cryptography"
+      }
+    },
+    {
+      "Type": "OpenSslExecutor",
+      "Parameters": {
+        "Scenario": "DES-EDE3",
+        "CommandArguments": "speed -elapsed -seconds 100 des-ede3",
+        "PackageName": "openssl",
+        "Tags": "CPU,OpenSSL,Cryptography"
+      }
+    },
+    {
+      "Type": "OpenSslExecutor",
+      "Parameters": {
+        "Scenario": "AES-128-CBC",
+        "CommandArguments": "speed -elapsed -seconds 100 aes-128-cbc",
+        "PackageName": "openssl",
+        "Tags": "CPU,OpenSSL,Cryptography"
+      }
+    },
+    {
+      "Type": "OpenSslExecutor",
+      "Parameters": {
+        "Scenario": "AES-128-CFB",
+        "CommandArguments": "speed -elapsed -seconds 100 aes-128-cfb",
+        "PackageName": "openssl",
+        "Tags": "CPU,OpenSSL,Cryptography"
+      }
+    },
+    {
+      "Type": "OpenSslExecutor",
+      "Parameters": {
+        "Scenario": "AES-192-CBC",
+        "CommandArguments": "speed -elapsed -seconds 100 aes-192-cbc",
+        "PackageName": "openssl",
+        "Tags": "CPU,OpenSSL,Cryptography"
+      }
+    },
+    {
+      "Type": "OpenSslExecutor",
+      "Parameters": {
+        "Scenario": "AES-192-CFB",
+        "CommandArguments": "speed -elapsed -seconds 100 aes-192-cfb",
+        "PackageName": "openssl",
+        "Tags": "CPU,OpenSSL,Cryptography"
+      }
+    },
+    {
+      "Type": "OpenSslExecutor",
+      "Parameters": {
+        "Scenario": "AES-256-CBC",
+        "CommandArguments": "speed -elapsed -seconds 100 aes-256-cbc",
+        "PackageName": "openssl",
+        "Tags": "CPU,OpenSSL,Cryptography"
+      }
+    },
+    {
+      "Type": "OpenSslExecutor",
+      "Parameters": {
+        "Scenario": "AES-256-CFB",
+        "CommandArguments": "speed -elapsed -seconds 100 aes-256-cfb",
+        "PackageName": "openssl",
+        "Tags": "CPU,OpenSSL,Cryptography"
+      }
+    },
+    {
+      "Type": "OpenSslExecutor",
```

Comment:
If I remember correctly CBC algorithms have known security vulnerabilities, could you do a little research on that?